### PR TITLE
create and edit README and MIT Licence files

### PR DESCRIPTION
**Pull request by C-Lion**
Summary
Fixes # Add README #3

List of changes proposed in this PR (pull-request)

- [x ] Add README.md 

- [ x] Add LICENCE.txt 

What should a reviewer concentrate their feedback on?

- [x ]  Created README with the standard sections. Added content per what I see in the repo. Of course will need updates as the project progresses but this gives you a framework to go with for now. Let me know if you wanted something else. 
- [x ] LICENCE.txt is just the standard MIT license from https://choosealicense.com/licenses/mit/#

 Everything looks ok?
Acknowledging contributors

- [ ]  All contributors to this pull request are already named in the table of contributors in the README file.
- [ v] The following people should be added to the table of contributors in the README file: c-lion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+```python
+python -m venv venv
+
+venv\Scripts\python.exe -m pip install -r requirements.txt
+```
+
+## Usage
+
+Under development.
+```
```
Comment (Fasten90) [inline README.md:18]: Can you finish or remove this 'code block' ?

The content at below shall not put into code block.

**Review by Fasten90 — CHANGES_REQUESTED** [review]
Thank you so much your contribution!  
I put a remark, if you can check.

**Comment by C-Lion** [comment]
@Fasten90 Is there any way you can please add add the 'hacktoberfest-accepted' label to the PR? 
(for reference: https://hacktoberfest.digitalocean.com/resources/participation)
Much appreciated.

**Comment by Fasten90** [comment]
Dear @C-Lion ,
I have put 'hacktoberfest-accepted' label to your merge request. #3 
Is it OK?
Thank you again your contribution.

BR

**Comment by C-Lion** [comment]
I think it needs to be on the PR not the issue. (A repository is considered to be participating in Hacktoberfest if it has the 'hacktoberfest' topic. If a repository isn't participating, an individual PR can be marked as participating by adding the 'hacktoberfest-accepted' label to it. ) Thanks

**Comment by Fasten90** [comment]
Sorry, you are right. I confused with the PR vs issue. Corrected. Please check :D

**Pull request merged**